addons/l10n_in/models/chart_template.py

```diff
@@ -17,6 +17,11 @@ def _prepare_all_journals(self, acc_template_ref, company, journals_dict=None):
                     journal['name'] = _('Tax Invoices')
         return res
 
+    def _load(self, sale_tax_rate, purchase_tax_rate, company):
+        res = super(AccountChartTemplate, self)._load(sale_tax_rate, purchase_tax_rate, company)
+        if self == self.env.ref("l10n_in.indian_chart_template_standard"):
+            company.write({'fiscalyear_last_month': '3'})
+        return res
```

Comment:
2 empty lines missing before the next class definition

Reply:
allo? @msh-odoo

Reply:
@jva-odoo

Reply:
@jco-odoo okay I take care of this